index.html

```diff
@@ -77,13 +77,15 @@ <h2>Contact</h2>
         Catering Service, 42nd Living Street, 43043 New York, NY
       </p>
       <p class="contactText">
-        You can also contact us by phone 00553123-2323 or email catering@catering.com, or you can send us a message here.
+        You can also contact us by phone <span>00553123-2323</span>  or email <span>catering@catering.com</span>, or you can send us a message here.
```

Comment:
No need to add tags in between without any specific purpose. Modify this to make phone number and email ID as links.